src/app/webcontainer/WebappSettingsPage.qml

```diff
@@ -202,6 +203,17 @@ FocusScope {
                                 dialog.confirmed.connect(clearCache);
                             }
                         }
+
+                        ListItem {
+                            objectName: "privacy.clearAllCookies"
+                            ListItemLayout {
+                                title.text: i18n.tr("Clear all cookies")
```

Comment:
This option is sentence cased but the two above it are Title Cased. Shall we pick one or the other?

Reply:
hmm, is "Clear All Cookies" ok ?

Reply:
On the other hand: the first 4 items are not Title Cased (Only allow ..., Domain blacklist/whitelist, Domain specific settings, Camera & microphone)
maybe it is easier to make it "Clear browsing history" and "Clear cache"
It is used in the question for the user, too.